galeracluster/source/documentation/galera-parameters.rst

```diff
@@ -1476,6 +1501,47 @@ This status variable is used to check which gmcast protocol version is used.
 This variable is mostly used for troubleshooting purposes and should not be implemented in a production environment.
 
 
+.. _`innodb_flush_log_at_trx_commit`:
+.. rst-class:: section-heading
+.. rubric:: ``innodb_flush_log_at_trx_commit``
+
+.. index::
+   pair: wsrep Provider Options; innodb_flush_log_at_trx_commit
+
+This variable controls the durability/speed trade-off for commits.
+
+The possible values are:
+
+- ``0`` - Nothing is done on commit; rather the log buffer is written and flushed to the InnoDB redo log once a second. This gives better performance, but a server crash can erase the last second of transactions.
+- ``1`` - The log buffer is written to the InnoDB redo log file, and a flush to disk performed after each transaction. This is required for full ACID compliance. Used with ``sync_binlog=1`` provides the greatest level of fault tolerance.
+- ``2`` - The log buffer is written to the InnoDB redo log after each commit, but flushing takes place every ``innodb_flush_log_at_timeout``(by default once a second). The performance is better, but an operating system crash or a power outage can cause the last second's transactions to be lost.
+- ``3`` - Flush to disk at prepare and at commit. This option is slower and usually redundant. 
+
+Options 0 and 2 can be faster than 1 or 3.
+
+.. csv-table::
+   :class: doc-options
+
+   "Default Value", "1"
+   "Dynamic", "Yes"
+   "Initial Version", "1.0"
+
+The excerpt below is an example of how this Galera parameter might look in the configuration file:
+
+.. code-block:: ini
+
+   innodb_flush_log_at_trx_commit="1"
+
+This variable can also be set dynamically at runtime:
+
+.. code-block:: mysql
+
+   SET GLOBAL innodb_flush_log_at_trx_commit=1;
+
+If you set ``innodb_flush_log_at_trx_commit`` dynamically at runtime, its value will be reset the next time the server restarts. To make the value persist on restart, set it also in a configuration file.

```

Comment:
Add a note that if you use MySQL 8 and greater, you can also use SET PERSIST to ensure the value persists upon restart.

Reply:
Added